docs/getting-started/provider-playbook/technology-providers-implementation/ed-fi-api-fundamentals/enumerations-descriptors.md

```diff
@@ -0,0 +1,88 @@
+* **description**: the full definition for the value
+* **shortDescription**: a shorter version of the description, if needed. This is sometimes used to populate a drop-down lists in user-facing interfaces.
+* **effectiveBeginDate:** this is the date that the descriptor was first available
+* **effectiveEndDate:** this is the date that the descriptor value is retired, if a value has been retired, the client must look for a new value that looks appropriate or communicate with the hosting agency/organization for guidance
+
+## Descriptor References
+
+In transit, when submitted or retrieved as part of an Ed-Fi API resource transaction, only the **codeValue** and the **namespace** are returned. A Descriptor looks like this in the API JSON
+
+```text
+uri://ed-fi.org/AcademicSubjectDescriptor#Chemistry
+```
+
+Values _must_ be sent as-is and _must not_ be URI or otherwise encoded. For example, a Descriptor whose codeValue has spaces _must_ be sent thus:
+
+```text
+uri://ed-fi.org/AcademicSubjectDescriptor#English Language Arts
+```
+
+   ...and _must not_ be sent as
+
+```text
+uri://ed-fi.org/AcademicSubjectDescriptor#English%20Language%20Arts
+```
+
+## Descriptor API Resources
+
+Many APIs (including that on the Ed-Fi ODS/API) provide lookup features for Descriptors. This allows an API to load and inspect all possible values for a particular enumeration. The API resource format for a Descriptor has this format:
+
+```json
+  {
+    "id": "47721c744e566176ad035dc8e119a2fb",
+    "codeValue": "English Language Arts",
+    "description": "English Language Arts",
+    "namespace": "uri://ed-fi.org/AcademicSubjectDescriptor",
+    "shortDescription": "English Language Arts"
+  }
+```
+
+Note that a Resource ID ("id") is provided here, consistent with other Ed-Fi REST API resources.
+
+## Ed-Fi, Provider, and Agency Values: What to Expect and What to Use
+
+Interoperability is complex, and enumeration values are part of this complexity. While it seems like the goal should be to coordinate the whole K-12 ecosystem to employ use a single set of values for each core K-12 concept, the reality is that education agencies and technology providers must adapt values to local needs and conditions.
+
+For agencies, such adaptation often reflects important local programs and priorities. Further, use of some sets of values might be mandated by (or reflect) state or local policy. These make adoption of a global set problematic.
+
+For their part, technology providers often have highly refined sets of values that reflect years worth of process and analytic investment.
+
+As a result, users of Ed-Fi APIs should expect to see enumeration sets with mixed namespaces.
+
+### Use Case-Specific Information
+
+Ed-Fi API specifications and certifications capture current community expectations as to what enumeration sets to use. Specifications in use can be found here:
+
+[Ed-Fi Data Exchange Standards Home](https://edfi.atlassian.net/wiki/spaces/EFDS/overview)
```

Comment:
I'm not sure what to do about this. These documents are ancient and should not be used - they refer to versions of the Data Standard that are no longer supported.

The RFC Home below are similarly not very useful as far as I'm concerned, because so out of date.

For now I'll leave these Confluence links in place, though I really don't want a technology provider to read them.